src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/commands.py

```diff
@@ -3,24 +3,44 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-#pylint: disable=line-too-long
+# pylint: disable=line-too-long
 
 from azure.cli.core.commands import cli_command
 from azure.cli.command_modules.documentdb._client_factory import (cf_documentdb)
+from ._client_factory import get_document_client_factory
+from ._command_type import cli_documentdb_data_plane_command
```

Comment:
This will cause the CLI to be slow to load because...
`._command_type` imports `.custom` which imports your SDK `azure.mgmt.documentdb.models`.
Importing of SDKs is slow and not required for loading the commands.

Can fix this by moving the import of `from .custom import generic_exception_handler` at `_command_type.py` to inside `def cli_documentdb_data_plane_command`